src/TMDBApp.Core/ViewModels/Base/ExtendedBindableObject.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Linq.Expressions;
+using System.Reflection;
+using Xamarin.Forms;
+
+namespace TMDBApp.Core.ViewModels.Base
+{
+    public abstract class ExtendedBindableObject : BindableObject
```

Comment:
Could have used INotifyPropertyChanged to avoid creating a dependency on Xamarin.Forms.